.github/workflows/test.yaml

```diff
@@ -76,4 +76,3 @@ jobs:
         brownie test ./tests/test_revoke.py -s
         brownie test ./tests/test_rewards.py -s
         brownie test ./tests/test_shutdown.py -s
-        brownie test ./tests/linkUsdc/test_happy_path.py -s
```

Comment:
plz add tests here.

Reply:
Added in c7998cda4e96167c953ec55f2661b8c397e2775f and 77bc8c7c76c1226585dc5252c09a4af65acccab5.